### PR TITLE
Change license to BSD and add copyrigt&license to source files

### PR DESCRIPTION
**Pull request by piyueh**
Changes according to #121

### CODE REVIEW

File: doc/Doxyfile [hunk 1]
```diff
@@ -228,7 +228,7 @@ TAB_SIZE               = 2
 # "Side Effects:". You can put \n's in the value part of an alias to insert
 # newlines.
 
-ALIASES                = 
+ALIASES                += license="@par License:^^"
```
Comment (mesnardo) [inline doc/Doxyfile]: On Phantom, when I open the API documentation and go to a source file, the license appears as:
`License:^^ BSD 3-Clause License.`

But if you change the ALIAS to:
`ALIASES                += license="@par License\n"`
then, it matches the copyright style.

Comment (piyueh) [reply doc/Doxyfile]: What version of Doxygen are you using? I'm using 1.8.14, and only `license=@par License:^^` works for me. The escape sequence `\n` doesn't work on my machine.

Comment (mesnardo) [reply doc/Doxyfile]: Interesting... on my mac, I have Doxygen 1.8.14 and the line return works as expected.
On Phantom, with Doxygen 1.8.11, there is no line return, it just displays the `^^`.
So let's keep the way you wrote it, but can you suppress the `:` to make it consistent with `Copyright` and `See also`?

Comment (piyueh) [reply doc/Doxyfile]: Done.

**Pull request merged**